.github/workflows/container.yml

```diff
@@ -22,12 +22,6 @@ jobs:
         shell: bash
         run: echo "GITHUB_BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/} | tr / -)" >> $GITHUB_ENV
 
-      # To build multi-arch images, see also: https://github.com/redhat-actions/buildah-build#multi-arch-builds
-      - name: Install qemu dependency
-        run: |
-          sudo apt-get update -y
-          sudo apt-get install -y binfmt-support qemu-user-static
```

Comment:
@kakkoyun FYI `qemu` does not appear to be setup in the `release` workflow, the release of an arm64 image will likely fail without merging the PR or adding it

Reply:
Thanks for pointing it out. We might have broken the release action since we haven't released anything after ARM64 arch support. I'll create a pre-release and test it.